x-pack/plugin/security/src/test/java/org/elasticsearch/xpack/security/transport/nio/SSLDriverTests.java

```diff
@@ -22,15 +23,25 @@
 import java.nio.Buffer;
 import java.nio.ByteBuffer;
 import java.nio.charset.StandardCharsets;
+import java.security.AccessController;
+import java.security.PrivilegedAction;
 import java.security.SecureRandom;
 import java.util.Arrays;
 import java.util.Collections;
 import java.util.List;
 import java.util.concurrent.atomic.AtomicInteger;
 import java.util.function.IntFunction;
 
+import static org.hamcrest.Matchers.is;
+
 public class SSLDriverTests extends ESTestCase {
 
+    @Before
+    public void setup() {
+        AccessController.doPrivileged((PrivilegedAction<String>) () -> System.setProperty("jdk.tls.disabledAlgorithms",
```

Comment:
Why do we disable TLSv1.1 in all JDK versions here ?

Reply:
Oops, that's a mistake from a previous approach. I've removed it.